sql/search_derived/mobile_search_clients_daily_v1/mobile_search_clients_daily.sql.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""
+Generate mobile search clients_daily query by creating a combined CTE for
+metrics and baseline for all Firefox Android apps, then print query to stdout
+"""
+import os
+
+APP_CHANNEL_TUPLES = [
```

Comment:
https://github.com/mozilla/bigquery-etl/issues/947 may be relevant here.

Reply:
That udf does look like it would simplify things here.  I'll watch for that change

Reply:
Feel free to create the UDF!